astro_2.0/features/AppHealth/AppHealth.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useRef, VFC } from 'react';
+import { useSocket } from 'context/SocketContext';
+import { NOTIFICATION_TYPES, showNotification } from 'features/notifications';
+
+export const AppHealth: VFC = () => {
+  const { socket } = useSocket();
+  const notyRef = useRef<boolean>(false);
+
+  useEffect(() => {
+    let intervalId: number;
```

Comment:
Looks like the variable is not being used.

Reply:
fixed